docs/packages/pkg/nng.rst

```diff
@@ -0,0 +1,20 @@
+.. spelling::
+
+    nng
+
+.. index::
+  single: unsorted ; nng
+
+.. _pkg.nng:
+
+nng
+===
+
+-  `Official <https://nanomsg.github.io/nng/index.html>`__
+-  `Example <https://github.com/cpp-pm/hunter/blob/master/examples/nng/CMakeLists.txt>`__
+-  Added by `tnixeu <https://github.com/tnixeu>`__ (`pr-N <https://github.com/cpp-pm/hunter/pull/N>`__)
```

Comment:
I personally don’t see much value in this line in a world with git blame but officially this PR number should be filled in

Reply:
Ok, i'll remove it.